active_record_shards.gemspec

```diff
@@ -6,8 +6,8 @@ Gem::Specification.new "active_record_shards", "3.4.3" do |s|
   s.description = "Easily run queries on shard and slave databases."
   s.license     = "MIT"
 
```

Comment:
add ruby version >= 2.1 since this is using prepend without send ... or use with send and make it 2.0